src/pybind/mgr/dashboard/frontend/src/app/shared/components/carbon-notification-panel/carbon-notification-panel.component.html

```diff
@@ -0,0 +1,89 @@
+<div class="cds--side-panel" [class.active]="isOpen">
+  <header class="cds--side-panel__header">
```

Comment:
There is  header [component](https://github.com/carbon-design-system/carbon-components-angular/blob/v5.56.2/src/ui-shell/header/header.component.ts)